mapApp/static/mapApp/js/tooltips.js

```diff
@@ -1,9 +0,0 @@
-$(function(){
```

Comment:
Can this file be deleted? It looks like it's empty.

Reply:
yes!